examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -108,6 +123,47 @@ class _GalleryAppState extends State<GalleryApp> {
 
   @override
   Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: _options.theme.data.copyWith(platform: _options.platform),
+      title: 'Flutter Gallery',
+      color: Colors.grey,
+      showPerformanceOverlay: _options.showPerformanceOverlay,
+      checkerboardOffscreenLayers: _options.showOffscreenLayersCheckerboard,
+      checkerboardRasterCacheImages: _options.showRasterCacheImagesCheckerboard,
+      routes: _buildRoutes(),
+      builder: (BuildContext context, Widget child) {
+        return Directionality(
+          textDirection: _options.textDirection,
+          child: _applyTextScaleFactor(child),
+        );
+      },
+      home: FutureBuilder<bool>(
+        future: _checkWelcomeFuture,
+        builder: (BuildContext context, AsyncSnapshot<bool> snapshot) {
+          if (snapshot.connectionState == ConnectionState.done) {
+            _showWelcome ??= snapshot.data;
+            return _buildHomeWidget(context);
+          } else {
+            return Container();
+          }
+        },
+      ),
+    );
+  }
+
+  Future<bool> _checkWelcomeState() async {
```

Comment:
Don't make the verb and the future result named so similarly.

Reply:
Renamed method